eftest/src/eftest/runner.clj

```diff
@@ -48,7 +48,7 @@
         (with-bindings* bindings f)))))
 
 (defn- default-thread-count []
-  (+ 2 (.availableProcessors (Runtime/getRuntime))))
+  (.availableProcessors (Runtime/getRuntime)))
```

Comment:
Why is this change necessary?

Reply:
I *presumed* this was an attempt to account for namespace threads.  I can revert if this is not true.

Reply:
It uses the same calculation as Clojure does. My assumption is that it does this because by adding 2, you guarantee that there are at least that number of threads. It might be a protection against an incorrect value returned by `.availableProcessors`.